self-core-impl/src/main/java/com/selfxdsd/core/BaseSelf.java

```diff
@@ -167,6 +167,16 @@ public ApiTokens apiTokens() {
             return this.user.apiTokens();
         }
 
+        @Override
+        public ApiToken register(
```

Comment:
@andreoss This method is not needed here. We will register the new token through the User, like this:

```java
final User authenticated = this.sell.login(Login);
final ApiToken registered = authenticated.register(name, token, expiration);
```